src/features/hotspots/settings/discovery/DiscoveryMap.tsx

```diff
@@ -217,7 +217,7 @@ const DiscoveryMap = ({
         zoomEnabled={false}
         onDidFinishLoadingMap={setupMap}
       >
-        <MapboxGL.Camera ref={cameraRef} />
+        <MapboxGL.Camera ref={cameraRef} zoomLevel={12} />
```

Comment:
There is also a `defaultSettings` zoomLevel, I think in Map.tsx I needed that to get it to load at the correct zoom level. Maybe this works though if it doesnt change?